specification/2.0/README.md

```diff
@@ -142,7 +142,7 @@ glTF has been designed to meet the following goals:
 * *Complete 3D scene representation.* Exporting single objects from a modeling package is not sufficient for many applications. Often, authors want to load entire scenes, including nodes, transformations, transform hierarchy, meshes, materials, cameras, and animations into their applications. glTF strives to preserve all of this information for use in the downstream application.
 * *Extensibility.* While the initial base specification supports a rich feature set, there will be many opportunities for growth and improvement. glTF defines a mechanism that allows the addition of both general-purpose and vendor-specific extensions.
 
-The design of glTF takes a pragmatic approach. The format is meant to mirror the GPU APIs as closely as possible, but if it did only that, there would be no cameras, animations, or other features typically found in both modeling tools and runtime systems, and much semantic information would be lost in the translation. By supporting these common constructs, glTF content can not only load and render, but it can be immediately usable in a wider range of applications and require less duplication of effort in the content pipeline.
+The design of glTF takes a pragmatic approach. The format is meant to mirror the GPU APIs as closely as possible, but if it did only that, there would be no cameras, animations, or other features typically found in both modeling tools and runtime systems, and much semantic information would be lost in the translation. By supporting these common constructs, glTF content can not only be loaded and rendered, but can be immediately usable in a wider range of applications and require less duplication of effort in the content pipeline.
```

Comment:
I feel like this part about "meant to mirror the GPU APIs as closely as possible" is a bit misleading; glTF is designed to be rendered via modern GPU APIs with as little data processing as possible, but isn't really a mirror of the APIs.

Reply:
The only change on this line was "glTF content can not only load and render, but it can be" -> "glTF content can not only be loaded and rendered, but can be".